tracing/src/main/java/io/airlift/tracing/OpenTelemetryModule.java

```diff
@@ -27,11 +27,13 @@
 import static com.google.common.base.Strings.nullToEmpty;
 import static io.airlift.configuration.ConfigBinder.configBinder;
 import static io.airlift.tracing.Tracing.attribute;
+import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class OpenTelemetryModule
         implements Module
 {
+    private static final String NODE_ANNOTATION_PREFIX = "io.airlift.node";
```

Comment:
just `airlift`?
or do we need any prefix at all?
since we pass-through user-provided annotations, we could as well just add them without any prefix.

Reply:
the link doesn't work for me

Reply:
Sorry, this one's correct: https://github.com/airlift/airlift/pull/1065#discussion_r1197221348